src/main/java/pl/sda/model/openWeatherAPI/Daily.java

```diff
@@ -0,0 +1,81 @@
+package pl.sda.model.openWeatherAPI;
+
+import java.time.LocalDateTime;
+
+public class Daily {
+    private long dt;
+    private Temp temp;
+    private int pressure;
+    private int humidity;
+    private float wind_speed;
```

Comment:
would be nice to have java proper convention (`windSpeed`) here (and configure gson to use `wind_speed`)